src/js/views/eva-variant-view.js

```diff
@@ -36,6 +36,83 @@ EvaVariantView.prototype = {
+            // along with the data item positions described here: http://www.insdc.org/files/feature_table.html#3.4.2
+            if (!(features_section_found || line.toLowerCase().startsWith("fh   key"))) {
+                      line_index += 1;
+                      continue;
+            }
+            features_section_found = true;
+            // Based on "Data item positions" described here, http://www.insdc.org/files/feature_table.html#3.4.2
+            // the sixth character represents the start of the feature key
+            if (!(source_line_found || line.slice(5,11).toLowerCase().startsWith("source"))) {
+                      line_index += 1;
+                      continue;
+            }
+            source_line_found = true;
+            if (line.slice(21,22) === "/") {
+              var assembled_line = line.trim();
+              line_index += 1;
+              // Assemble text spread across multiple lines until
+              // we hit the next qualifier (starts with /) or the next section
+              while (line_index < num_lines && !(response_lines[line_index].slice(21,22) === "/" || response_lines[line_index].slice(5,6).trim() !== "")) {
+                line = response_lines[line_index];
+                assembled_line += " " + line.slice(21).trim();
+                line_index += 1
+              }
+              var chromosome_regexp = new RegExp('.*/chromosome=".+"', 'g');
+              var organelle_regexp = new RegExp('.*/organelle=".+"', 'g');
+              var note_regexp = new RegExp('.*/note=".+"', 'g');
+              chosen_response = chromosome_regexp.exec(assembled_line) || organelle_regexp.exec(assembled_line) || note_regexp.exec(assembled_line);
+              if (chosen_response || line.slice(5,6).trim() !== "") {
+                break;
+              }
+            }
+            else {
+              line_index += 1;
+            }
+          }
+          if (!chosen_response || chosen_response.length == 0) {
+              return "";
+          }
+          return chosen_response[0].split('"')[1].trim();
+    },
+
+    get_chromosome_number_for_accession_with_retries: function(chromosome_accession) {
+        // Cache the contig resolution because this function could be called
+        // multiple times for the same contig in case of multi-allelic alleles (ex: ss1996903386)
+        if (chromosome_accession in this.chromosome_contig_map) {
+            return this.chromosome_contig_map[chromosome_accession];
+        }
+
+        this.chromosome_contig_map[chromosome_accession] =
+                (this.get_chromosome_number_for_accession(chromosome_accession, 1000) ||
+                this.get_chromosome_number_for_accession(chromosome_accession, 10000) ||
+                this.get_chromosome_number_for_accession(chromosome_accession, 100000));
```

Comment:
Retrieving such results where the chromosome information appears after 100k lines can take too long and might not be worth the trouble. In the worst case, it can lock up the users' browser particularly if they have under-powered machines. I ran the above logic against all the NCBI imported species and did not have any issues resolving the contigs. See [comment](https://www.ebi.ac.uk/panda/jira/browse/EVA-1664?focusedCommentId=292533&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-292533)